app.py

```diff
@@ -76,7 +81,7 @@ def signup():
             user = db.query(User).filter(User.name == form.name).first()
             user.generate_email_confirmation_token()
             user.send_confirmation_email(mail)
-            return "user {} created".format(user.name)
+            return render_template("confirm_email.html", user=user)
 
     else:
         return render_template("landing.html", form=form, role=session['role'])
```

Comment:
роль уже не актуально, из-за этого ошибка, скорее всего
```suggestion
        return render_template("landing.html", form=form)
```